test/engine/upsert.result

```diff
@@ -1585,21 +1585,29 @@ s:upsert({0}, {{'!', 42, 42}})
 ...
 s:select{0}
 ---
-- - [0, 1, 2]
+- - [0, 1, 2, null, null, null, null, null, null, null, null, null, null, null, null,
+    null, null, null, null, null, null, null, null, null, null, null, null, null,
+    null, null, null, null, null, null, null, null, null, null, null, null, null,
+    42]
 ...
 s:upsert({0}, {{'#', 42, 42}})
 ---
 ...
 s:select{0}
 ---
-- - [0, 1, 2]
+- - [0, 1, 2, null, null, null, null, null, null, null, null, null, null, null, null,
+    null, null, null, null, null, null, null, null, null, null, null, null, null,
+    null, null, null, null, null, null, null, null, null, null, null, null, null]
 ...
 s:upsert({0}, {{'=', 42, 42}})
 ---
 ...
 s:select{}
 ---
-- - [0, 1, 2]
+- - [0, 1, 2, null, null, null, null, null, null, null, null, null, null, null, null,
```

Comment:
I think the purpose of the test with setting a value with an index out of range was to ensure it does not work. Now it works, so better just drop these `upsert()` with 42th field.